_posts/2023-09-24-update.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Week Ending September 24, 2023
+date: 2023-09-24 22:00:00 -0000
+slug: 2023-09-24-update
+---
+
+## Developer News
+
+
+## Release Schedule
+
+**Next Deadline: PRR Freeze, September 28th (this Thursday)**
+
+Get your KEPs ready for Production Readiness Review. Opt-in now if you’re going to have the feature in 1.29.
+September Patch Release cherry-picks are due this Friday (September 29th).
+
+## Featured PRs
+
+
+## KEP of the Week
+
+
+## Other Merges
+*
+*
+
+
+## Promotions
```

Comment:
Not trying to step on any toes here, but as someone who works with product marketing, some folks will see this as "Promotion" in a "Oh can I buy a spot in this newsletter?" kind of way. Might want to say Software/API/etc. in front of it.

Reply:
We've been using that section name for 3 years now.  I don't think this is sufficiently enough of an issue to warrant changing it.